pkg/acsengine/defaults.go

```diff
@@ -590,8 +585,8 @@ func setVMSSDefaults(a *api.Properties) {
 	}
 }
 
-// SetAgentNetworkDefaults for agents
-func setAgentNetworkDefaults(a *api.Properties, isUpgrade, isScale bool) {
+// setAgentProfileDefaults sets defaults for agent profiles
```

Comment:
Renamed this func because it's not network-specific, it's agent profile-specific.